spec/controllers/items_controller_spec.rb

```diff
@@ -245,4 +259,8 @@ def put_update(item_id, wishlist_id, item_params)
     put :update,
       params: { id: item_id, wishlist_id: wishlist_id, item: attributes_for(:item, item_params) }
   end
+
+  def delete_destroy(item_id:, wishlist_id:)
```

Comment:
I've seen this before, and I guess I didn't comment on this. I know these arguments are named, and can be used in any order we want, but the actual hierarchy of resources is `wishlist > item` and I'd keep it here as well. BUt that's just a tidbit you can ignore.